PWGGA/Hyperon/AliSigma0PhotonMotherCuts.cxx

```diff
@@ -364,7 +382,10 @@ void AliSigma0PhotonMotherCuts::SigmaToLambdaGammaMixedEvent(
         fHistMixedInvMassPt->Fill(pT, invMass);
         if (!fIsLightweight) {
```

Comment:
The histograms are initialised anyways, so could you fill them all the time (not only when fIsLightweight is true